Multi-line string literals are treated as errors in gcc 3.3 and later.
Ideally we'd use ANSI-style string concatenation instead, but compile_et
neither knows how to convert the strings nor how to pass them through from
its input files.  Aaargh!

--- krb5-1.3/src/kadmin/passwd/kpasswd_strings.et
+++ krb5-1.3/src/kadmin/passwd/kpasswd_strings.et
@@ -23,8 +23,7 @@
 
 # /* M5 */
 error_code KPW_STR_CANT_OPEN_ADMIN_SERVER,
-"Cannot establish a session with the Kerberos administrative server for\n\
-realm %s. %s."
+"Cannot establish a session with the Kerberos administrative server for\nrealm %s. %s."
 # /* <realm-name>, <Specific error message from admin server library>. */
 
 # /* M6 */
--- krb5-1.3/src/lib/kadm5/chpass_util_strings.et
+++ krb5-1.3/src/lib/kadm5/chpass_util_strings.et
@@ -16,34 +16,21 @@
 error_code CHPASS_UTIL_PASSWORD_CHANGED, "Password changed.\n"
 
 error_code CHPASS_UTIL_PASSWORD_IN_DICTIONARY,
-"New password was found in a dictionary of possible passwords and\n\
-therefore may be easily guessed. Please choose another password.\n\
-See the ovpasswd man page for help in choosing a good password."
+"New password was found in a dictionary of possible passwords and\ntherefore may be easily guessed. Please choose another password.\nSee the ovpasswd man page for help in choosing a good password."
 
 error_code CHPASS_UTIL_PASSWORD_NOT_CHANGED, "Password not changed."
 
 error_code CHPASS_UTIL_PASSWORD_TOO_SHORT, 
-"New password is too short.\n\
-Please choose a password which is at least %d characters long."
+"New password is too short.\nPlease choose a password which is at least %d characters long."
 #	/*  <pw-min-len> */
 
 error_code CHPASS_UTIL_TOO_FEW_CLASSES, 
-"New password does not have enough character classes.\n\
-The character classes are:\n\
-	- lower-case letters,\n\
-	- upper-case letters,\n\
-	- digits,\n\
-	- punctuation, and\n\
-	- all other characters (e.g., control characters).\n\
-Please choose a password with at least %d character classes."
+"New password does not have enough character classes.\nThe character classes are:\n	- lower-case letters,\n	- upper-case letters,\n	- digits,\n	- punctuation, and\n	- all other characters (e.g., control characters).\nPlease choose a password with at least %d character classes."
 # /* <min-classes> */
 
 
 error_code CHPASS_UTIL_PASSWORD_TOO_SOON,
-"Password cannot be changed because it was changed too recently.\n\
-Please wait until %s before you change it.\n\
-If you need to change your password before then, contact your system\n\
-security administrator."
+"Password cannot be changed because it was changed too recently.\nPlease wait until %s before you change it.\nIf you need to change your password before then, contact your system\nsecurity administrator."
 # /* <ctime(last-pw-change+pw-min-life)> */
 
 error_code CHPASS_UTIL_PASSWORD_REUSE, 
